specs/csi-plug-n-play/Readme.md

```diff
@@ -0,0 +1,71 @@
+# SODA-CSI Plug-N-Play
```

Comment:
Original file history lost.
Can we make the design document as a design document name?
Also the alternate, if you want a separate file, please name it accordingly and link. Need to cross link both doc.
Else, update the original and avoid the duplicate information. (intro, goals etc...)